fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/ambassador/TransmissionSimulator.java

```diff
@@ -41,7 +41,7 @@
 
 public class TransmissionSimulator {
 
-    private final static Logger log = LoggerFactory.getLogger(TransmissionSimulator.class);
+    private final static Logger log = LoggerFactory.getLogger(SnsAmbassador.class);
```

Comment:
I'd prefer that log messages generated in this class will also have the class name as prefix. Was this proposed by @rprotzmann ?

Reply:
Exactly, @rprotzmann requested this change in our yesterday's meeting.

Reply:
Yes, I would prefer this solution - otherwise sendV2XMessage is logged with TransmissionSimulator (which actually calculated the whole transmission from sending to receiving), but recvV2XMessage is logged again by SnsAmbassador (because it is previously splitted into several invidual interactions